packages/client-library-otel/src/instrumentation.ts

```diff
@@ -264,3 +266,26 @@ function mergeConfigs(
     monitor: Object.assign(fallbackConfig.monitor, userConfig?.monitor),
   };
 }
+
+/**
+ * We aren't getting access to the FPX_ENDPOINT in the fetch function in Node,
+ * this helper will also check process.env
+ *
+ * TODO - Try using `env` from hono/adapter to get the env var?
+ */
+function getFpxEndpoint(env: HonoLikeEnv) {
+  const fpxEndpointFromEnv =
+    typeof env === "object" && env !== null
+      ? (env as Record<string, string | null>).FPX_ENDPOINT
+      : null;
+
+  if (fpxEndpointFromEnv) {
+    return fpxEndpointFromEnv;
+  }
+
+  if (typeof process !== "undefined" && typeof process.env !== "undefined") {
```

Comment:
todo: factor this out into a smarter helper, or do some basic runtime detection like we did in autometrics